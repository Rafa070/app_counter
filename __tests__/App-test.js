import renderer from 'react-test-renderer';
import App from '../App';



describe('Teste de Snapshot', () => {
    it('Snapshot de App', () => {
        const tela = renderer.create(<App/>).toJSON();
        expect(tela).toMatchSnapshot();
    })
})
