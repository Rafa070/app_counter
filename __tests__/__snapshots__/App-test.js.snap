// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de Snapshot Snapshot de App 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      collapsable={false}
      forwardedRef={[Function]}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="rgb(255, 255, 255)"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={true}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="Welcome"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      />
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            Object {
              "backgroundColor": "rgb(242, 242, 242)",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "flex-start",
              "backgroundColor": "rgba(234,234,234,1)",
              "display": "flex",
              "flexDirection": "column",
              "height": 926,
              "justifyContent": "flex-start",
              "paddingBottom": 166,
              "paddingLeft": 40,
              "paddingRight": 40,
              "paddingTop": 203,
              "width": 428,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
              }
            }
          >
            <Image
              source={
                Object {
                  "uri": "https://firebasestorage.googleapis.com/v0/b/unify-bc2ad.appspot.com/o/nu6yxatp2m-75%3A74?alt=media&token=33da3fdf-5561-441f-80e1-6569c84b68e4",
                }
              }
              style={
                Object {
                  "height": 226,
                  "marginBottom": 10,
                  "width": 295,
                }
              }
              testID="imageWelcome"
            />
            <Text
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                  "fontSize": 30,
                  "fontWeight": "800",
                }
              }
            >
              Let's Count Together
            </Text>
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "borderColor": "rgba(0,166,166,1)",
                  "borderRadius": 15,
                  "borderStyle": "solid",
                  "borderWidth": 2,
                  "display": "flex",
                  "flexDirection": "row",
                  "height": 48,
                  "justifyContent": "center",
                  "marginBottom": 10,
                  "marginTop": 35,
                  "opacity": 1,
                  "paddingBottom": 8,
                  "paddingLeft": 8,
                  "paddingRight": 8,
                  "paddingTop": 8,
                  "width": 320,
                }
              }
              testID="btnLogin"
            >
              <Text
                style={
                  Object {
                    "color": "rgba(67,66,66,1)",
                    "fontSize": 18,
                    "fontWeight": "800",
                  }
                }
              >
                Login
              </Text>
            </View>
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgba(0,166,166,1)",
                  "borderRadius": 15,
                  "display": "flex",
                  "flexDirection": "row",
                  "height": 48,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 9,
                  "paddingLeft": 9,
                  "paddingRight": 9,
                  "paddingTop": 9,
                  "width": 320,
                }
              }
              testID="btnCadastro"
            >
              <Text
                style={
                  Object {
                    "color": "black",
                    "fontSize": 18,
                    "fontWeight": "800",
                  }
                }
              >
                Cadastro
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
